translations/es.json

```diff
@@ -22,7 +22,9 @@
       "nick": "apodo",
       "input": "mensaje",
       "search": "buscar por etiqueta",
-      "aria": "Visita la sala de chat de datafruits.fm"
+      "aria": "Visita la sala de chat de datafruits.fm",
+      "coc": "Code of Contact",
+      "i_agree": "He leído el Código de Conducta y acepto seguir sus términos y entiendo que pueden ser removidos por no hacerlo."
```

Comment:
I couldn't find where this key was being used.

Reply:
And where is the English version ?

Reply:
Thanks for catching these, they aren't needed so I'll remove them